.packit.yaml

```diff
@@ -312,3 +312,24 @@ jobs:
       test:
         tmt:
           name: regression-operands
+  - job: tests
+    trigger: release
+    identifier: "performance"
+    targets:
+      - centos-stream-9-x86_64
+      - centos-stream-9-aarch64
+    skip_build: true
+    manual_trigger: true
```

Comment:
You can't have `trigger: release` and `manual_trigger: true` together. it will never get executed. Also you won't be able to run any on PRs, just on releases and I don't think it's a good idea.

Reply:
Updated ;)